mathesar_ui/src/components/abstract-type-control/AbstractTypeControl.svelte

```diff
@@ -80,9 +79,17 @@
     !selectedAbstractType ||
     typeChangeState?.state === 'processing' ||
```

Comment:
`typeChangeState` needs to be updated within `onSave` during errors or success, otherwise it'll always remain in processing, resulting in the save button being disabled.

Reply:
1caaaab9